packages/client/lib/functions/ProofUtil.ts

```diff
@@ -87,7 +87,7 @@ const createJWT = (jwtProps?: JwtProps, existingJwt?: Jwt): Jwt => {
   };
 
   const jwtHeader: JWTHeaderParameters = {
-    typ: 'JWT',
+    typ: 'openid4vci-proof+jwt',
```

Comment:
Please do not blindly update these types of values. This is obviously something which should depend on the version of the spec

Reply:
Fixed!